tests/integration/conftest.py

```diff
@@ -741,8 +760,9 @@ def cross_cdf_table2(engine, metadata):
 def pytest_addoption(parser):
     parser.addoption(
         "--backend",
-        choices=(("mssql", "mssql-freetds", "postgres", "snowflake")),
+        choices=(("mssql", "mssql-freetds", "postgres", "snowflake", "bigquery")),
         help="which database backend to use to run the integration tests",
+        default="bigquery",

```

Comment:
We don't play favorites :p

```suggestion

```

Reply:
that was wrongly added, will remove it. I couldn't find another way in vscode to set it....Maybe you know how to do it e.g. in lauch.json?

Reply:
I guessed it was part of your workflow. 

I never tried to customize how the tests are executed with the integrated test executioner in VSCode.